.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-config.js
\ No newline at end of file
+config.js
```

Comment:
rename config file to .config.js  and  make git ignore  file reflect this  by adding .config/js

Reply:
Yeah, as Joe already said, kindly rename it to **_.config_** from **_config_** and also update your server.js file

Reply:
Thank you Joe, added .config to git ignore.